src/test/ui/E0642.stderr

```diff
@@ -0,0 +1,23 @@
+error[E0642]: patterns aren't allowed in trait methods
+  --> $DIR/E0642.rs:12:12
+   |
+LL |     fn foo((x, y): (i32, i32)); //~ ERROR patterns aren't allowed in trait methods
+   |            ^^^^^^
+help: give this argument a name or use an underscore to ignore it, instead of a tuple pattern
+   |
+LL |     fn foo(_: (i32, i32)); //~ ERROR patterns aren't allowed in trait methods
+   |            ^
+
+error[E0642]: patterns aren't allowed in trait methods
+  --> $DIR/E0642.rs:16:12
+   |
+LL |     fn bar((x, y): (i32, i32)) {} //~ ERROR patterns aren't allowed in trait methods
+   |            ^^^^^^
+help: give this argument a name or use an underscore to ignore it, instead of a tuple pattern
```

Comment:
reword to "give this argument a name or use an underscore to ignore it instead of using a tuple pattern"